layer/USAGE_android.md

```diff
@@ -1,143 +1,151 @@
+Log Output To Console | debug.gfxrecon.log_output_to_console | BOOL | Log messages will be written to Logcat.
+Memory Tracking Mode | debug.gfxrecon.memory_tracking_mode | STRING | Specifies the memory tracking mode to use for detecting modifications to mapped Vulkan memory objects. Available options are: "page_guard", "assisted" and "unassisted". Default is "page_guard". <ul><li>"page_guard" tracks modifications to individual memory pages, which are written to the capture file on calls to `vkFlushMappedMemoryRanges`, `vkUnmapMemory`, and `vkQueueSubmit`. Tracking modifications requires allocating shadow memory for all mapped memory.</li><li>"assisted" expects the application to call `vkFlushMappedMemoryRanges` after memory is modified; the memory ranges specified to the `vkFlushMappedMemoryRanges` call will be written to the capture file during the call.</li><li>"unassisted" writes the full content of mapped memory to the capture file on calls to `vkUnmapMemory` and `vkQueueSubmit`. It is very inefficient and may be unusable with real world applications that map large amounts of memory.</li></ul>
 
 ## Capture Files
+By default, capture files are written to `/sdcard/gfxrecon_capture.gfxr`
 
-By default, capture files are dumped out to the `/sdcard`, and will
-use the name `gfxrecon_capture.gfxr`:
+Capture files are created on the first call to vkCreateInstance, when the Vulkan
+loader loads the capture layer, and are closed on vkDestroyInstance, when the 
+last active instance is destroyed and the layer is unloaded.
 
- ```
-/sdcard/gfxrecon_capture.gfxr
-```
-
-Enabling timestamps may be needed to prevent apps that create multiple files from
-overwriting existing files.
-
-### Overriding the Default Name
+If multiple instances are created concurrently, only one capture file will be created.
+If multiple instances are created consecutively, each instance will create a new file.
+For applications that create multiple instances consecutively, it may be necessary to
+enable capture file timestamps to prevent each new instance from overwriting the file
+created by the previous instance.
 
-You may override the name and location of the capture file by setting the
-property `debug.gfxrecon.capture_file` to point to what you would like to
-output as defined above in [GFXRecon Debug Options](#gfxrecon-debug-options).
-
-This may be done by performed by using `adb shell` in the following way:
+### Specifying Capture File Location
+The capture file's save location can be specified by setting the
+`debug.gfxrecon.capture_file` system property, described above in
+the [GFXRecon Debug Options](#gfxrecon-debug-options) section.
 
+The property may be set through ADB with the following command:
 ```
-adb shell "setprop debug.gfxrecon.capture_file '<your_path_and_file_name_here>.gfxr'"
+adb shell "setprop debug.gfxrecon.capture_file '<path_to_file>.gfxr'"
```

Comment:
This implies the .gfxr suffix is required.  I'm not sure it is - is it?

Reply:
It isn't strictly required.